index.html

```diff
@@ -245,10 +245,10 @@
   </script>
   <script src="https://unpkg.com/tiny-markdown-editor/dist/tiny-mde.min.js"></script>
   <script type="text/javascript">
-    function initMarkdownEditor() {
-      let tinyMDE = new TinyMDE.Editor({ element: document.getElementById("editor") });
+    function initMarkdownEditor(editor, toolbar) {
```

Comment:
can we somehow make the editor component completely self-contained without requiring this global function defined here? even if it requires something like [inline js](https://rustwasm.github.io/docs/wasm-bindgen/reference/js-snippets.html#using-inline_js) in the rust code?

Reply:
Some like this?

```rust
#[wasm_bindgen(inline_js = r#"
    export function initMarkdownEditor(editor_id, toolbar_id, onChangeCallback) {
        let editorElement = document.getElementById(editor_id);
        let toolbarElement = document.getElementById(toolbar_id);
        
        let tinyMDE = new TinyMDE.Editor({ element: editorElement });
        let commandBar = new TinyMDE.CommandBar({
            element: toolbarElement,
            editor: tinyMDE,
        });

        tinyMDE.addEventListener('change', function () {
            let content = tinyMDE.getContent();
            onChangeCallback(content);
        });
    }

    export function setContentMarkdownEditor(editor_id, content) {
        let editorElement = document.getElementById(editor_id);
        let tinyMDE = new TinyMDE.Editor({ element: editorElement });
        tinyMDE.setContent(content);
    }
"#)]
```